app/components/login-form.hbs

```diff
@@ -25,7 +25,6 @@
           <input
             autocapitalize="off"
             autocorrect="off"
-            autofocus=""
```

Comment:
I think we can keep this [the docs](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-autofocus-attribute.md#no-autofocus-attribute) says we can use it on elements that 
> serve as the main purpose of the page

Reply:
:heavy_check_mark: